daml-assistant/get-daml.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# This script downloads and installs the DAML SDK on Linux and macOS.
+# This will overwrite any existing installation in ~/.daml
+#
+
+readonly ORIGDIR="$(pwd)"
+function cleanup() {
+  echo ""
+  echo "$(tput setaf 3)FAILED TO INSTALL!$(tput sgr 0)"
+  cd $ORIGDIR
+}
+trap cleanup EXIT
+
+readonly TMPDIR="$(mktemp -d)"
+cd $TMPDIR
+
+readonly DAML_HOME="$HOME/.daml"
+if [ -d $DAML_HOME ] ; then
+  chmod -R u+w $DAML_HOME
+  rm -rf $DAML_HOME
+fi
+
+
+echo "Determining latest DAML version..."
+readonly VERSION="$(curl -s https://github.com/digital-asset/daml/releases/latest | sed 's/^.*github.com\/digital-asset\/daml\/releases\/tag\/v//' | sed 's/".*$//')"
```

Comment:
I suggest adding a check if the `curl` tool exists to provide a quality error message

Reply:
Will do, thanks!